requirements.txt

```diff
@@ -1,2 +1,2 @@
 # Make sure clams-python version is explicitly specified, at least the lower bound
-clams-python==1.2.3
+clams-python==1.2.4
```

Comment:
now 1.2.5 is release, let's bump this up as well. (check `Containerfile` as well. )

Reply:
you need to update the base image version in containerfile as well. Also, there was a small bug in the SDK version that you used to generate the app template. line 28 of the Containerfile, 
https://github.com/clamsproject/app-text-slicer/blob/6253ad3c572e1116e029201a65178a023746efea/Containerfile#L28
the `&&` should be just `;`. So the line should be 
```
RUN mkdir /cache ; rm -rf /root/.cache ; ln -s /cache /root/.cache
```